modules/ROOT/pages/mule-concepts.adoc

```diff
@@ -1,14 +0,0 @@
-= Key Concepts
```

Comment:
This file was an orphaned page of links. Links to it have been removed or revised. If there are other links to it from outside this branch, a page alias for it is in index.adoc, which covers Mule concepts.